src/content/release-notes.mdx

```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
+- **Buildkit Persistence Enabled**: Buildkit persistence is now enabled by default, with a 100Gi disk and cache set to 90% of the disk size. If you previously used buildkit.persistence.cache, adjust to the new ratio, as this setting is no longer applicable.
+
+### New Features
+
+- **[Introducing the Okteto Resource Manager](admin/resource-manager.mdx)**: a new feature that automatically optimizes CPU and memory requests for your environments. By analyzing real-time resource utilization, the Resource Manager dynamically adjusts resource requests to ensure efficient usage, prevent node overload, and improve overall cluster performance. This feature simplifies resource management, reduces manual adjustments, and enhances application stability, especially in larger clusters. The default installation provides recommendations but doesn't apply them automatically. [See our docs for details on how to apply these automatically](admin/resource-manager.mdx).
+- Added Okteto [Garbage Collector settings to the Admin Dashboard](admin/cleanup.mdx): Admins can now manage sleep and delete periods from the Dashboard, with the ability to set different configurations for Namespaces and Preview Environments
+- [Remote Execution can now be made the default](core/remote-execution.mdx) by configuring it in the Okteto Admin Dashboard. This improves usability, particularly for private Git repositories and context synchronization with BuildKit, making remote deployments more consistent and reliable
+- Added additional feature and documentation for [running Buildkit at scale](self-hosted/manage/buildkit-high-performance.mdx)  <!-- 8727 -->
+
+### Improvements
+
+- Okteto will now automatically create a Docker secret if one doesn't already exist in the Controller Manager. This prevents a misleading warning that was being displayed in some scenarios when deploying from the UI  <!-- 8566 -->
+- [Buildkit cache size is now automatically configured](self-hosted/helm-configuration.mdx#buildkit) based on its PVC volume size  <!-- 8723 -->
+- Removed Buildkit persistency as an installation step now that it defaults to true  <!-- 8771 -->
+- The Okteto frontend now runs rootless by default, while Buildkit operates without privileges when rootless mode is enabled  <!-- 8774, 8800 -->
+- We've [released Okteto CLI 3.1.0 with many new improvements](https://github.com/okteto/okteto/releases/tag/3.1.0)
+- [Okteto CLI 3.1.0](https://github.com/okteto/okteto/releases/tag/3.1.0): [Added support for a `context`](reference/okteto-manifest.mdx#context-string-optional) field in the Okteto Manifest, allowing you to specify the working directory for commands in the `deploy` and `destroy` sections
```

Comment:
The link of context points to `test` section. I think we are missing an update of the `deploy` and `destroy` command to add the `context` property, so I'd remove the link for now. I had added the link on my previous message just to illustrate that the one we added was similar to the one we already have for `test`